### ISSUE TITLE
Language support

### ISSUE THREAD

**Issue by ahopkins** [sanic-org/sanic-guide#4]
We want to add support for language translations. Will fill out with more details.

@miss85246

**Comment by ZinkLu** [comment]
@ahopkins Hi,👋  pls add me to repo.

**Comment by ZinkLu** [comment]
@miss85246 I've created a new branch, Can you please add your `ZH` folder first? It seems that you had a proper vuepress configs.

**Comment by ZinkLu** [comment]
Should we had an issue only cares Simplified Chinese ?

**Comment by miss85246** [comment]
@ZinkLu of course, but if we only care Simplified Chinese in new branch, how can we deploy it ？
As far as I know, github pages can only be deployed once in a repo.
Do you have any good solutions?

**Comment by ZinkLu** [comment]
we can merge to master after we finish the translation weekly, what do you think?

**Comment by ahopkins** [comment]
I would suggest having a branch per page maybe, with separate PRs. Then you can pull locally to view/test. Once we get past the initial stage, hopefully it will not be too difficult to track the pages that need updates. Ideally all changes would have the translations.

**Comment by ZinkLu** [comment]
You mean having branches for each page with multiple languages? It sounds reasonable cause changes can be tracked easily on the very page.

**Comment by ZinkLu** [comment]
My first thought was to have a branch for each language, changes can be tracked by commits on master, though it's not very convenient

**Comment by ZinkLu** [comment]
we can use https://www.transifex.com/ for translation work too , I've participated in Gino document translation project on the platform before.

**Comment by ahopkins** [comment]
@ZinkLu how does that work? Is it a platform that we would write our documentation in? That does not sound so convenient since we would not have a running server alongside it.

**Comment by ZinkLu** [comment]
Okay, forget about the platform, It's a professional translation platform and needs couples of steps to import and export the docs. 

It's a good idea to have branches for each doc.  @miss85246 any thoughts?

**Comment by ahopkins** [comment]
A single branch for the initial is probably enough, right?

**Comment by miss85246** [comment]
I agree that every document has a branch. I am a little busy this weekend, and then I will try to create a branch for each document in my fork repo later, if this is feasible.

**Comment by ZinkLu** [comment]
@miss85246 Don't hurry, let’s talk it over on gitter, I’ve sent you a message.

**Comment by miss85246** [comment]
@ZinkLu : 😎  I have submitted a merge for simplified Chinese branch. You can have a look. If there is no problem, we will execute the merge operation.

@ahopkins : We discussed it for a while, and we think that after finishing the translation work at hand, we can create branches for each document and use a new organization form. what do you think ? 😃

**Comment by ahopkins** [comment]
Sounds good to me 👍

**Comment by ZinkLu** [comment]
Great work! I'll push mine tonight after replace some oral expressions.

**Comment by yaakiyu** [comment]
I can't translate on Crowdin because it says 'The project you are trying to translate is currently suspended.'
Will you fix it or explain why it is caused?

**Comment by ahopkins** [comment]
Because we never fully implemented it and I wasn't going to pay for something that wasn't in use. It seems our translations have fallen behind and we either need a new solution with renewed effort from translators, or we should take it down because it's getting out of date and difficult to manage.

**Comment by miss85246** [comment]
Honestly, we lack an effective management method.
After moving the new translation platform, I'm at a loss. 
Because the economy has gone from bad to worse this year, our work tasks have also increased a lot. 
Apart from work and rest, we have almost no time do translation work and think how to solve management problems.

**Comment by ahopkins** [comment]
@miss85246 i agree and understand. I meant no criticism. OSS contribution is all voluntary, which means people's interests and availability comes in and out. That's understandable and expected. But it also means that I cannot justify the expense of we will not keep up with it. At some point it is more harmful to have out of date docs than it is to have no docs.

**Comment by yaakiyu** [comment]
Crowdin changes the target sentences into the latest original ones when they are updated, even if there are already "old" translations. ([See here](https://github.com/sanic-org/sanic-guide/pull/131/commits/95d4b2e03366fddabf81fca0473db30c693e5401))
So there seem to be no problems in old translations.
I think we should marge the PR as often as possible.
